development/CohortOntology.owl

```diff
@@ -176,7 +185,15 @@
     <!-- https://w3id.org/enigma#hasCohortProject -->
 
     <owl:ObjectProperty rdf:about="https://w3id.org/enigma#hasCohortProject">
-        <rdfs:domain rdf:resource="https://w3id.org/enigma#Cohort"/>
+        <rdfs:domain>
+            <owl:Class>
+                <owl:unionOf rdf:parseType="Collection">
+                    <rdf:Description rdf:about="https://w3id.org/enigma#Cohort"/>
+                    <rdf:Description rdf:about="https://w3id.org/enigma#CohortGroup"/>
+                    <rdf:Description rdf:about="https://w3id.org/enigma#ImageAcquisitionProtocol"/>
```

Comment:
An image acquisitionProtocol hasCohortProject project??

Reply:
According to the spreadsheet, yes they wanted hasCohortProject applied to ImageAcquisition protocol

Reply:
Superweird, but ok...